site/content/contribute/mobile/developer-setup/_index.md

```diff
@@ -5,38 +5,42 @@ weight: 1
 subsection: Mobile Apps
 ---
 
-The following instructions apply to the mobile apps for iOS and Android built in React Native. Download the iOS version [here](http://about.mattermost.com/mattermost-ios-app/) and the Android version [here](http://about.mattermost.com/mattermost-android-app/). Source code can be found at https://github.com/mattermost/mattermost-mobile.
+The following instructions apply to the mobile apps for iOS and Android built in React Native. Download the iOS version <a href="http://about.mattermost.com/mattermost-ios-app/" target="_blank">here</a> and the Android version <a href="http://about.mattermost.com/mattermost-android-app/" target="_blank">here</a>. Source code can be found at the <a href="https://github.com/mattermost/mattermost-mobile" target="_blank">GitHub Mattermost Mobile app repository</a>.
```

Comment:
```suggestion
The following instructions apply to the mobile apps for iOS and Android built in React Native. Download the iOS version [here](http://about.mattermost.com/mattermost-ios-app) and the Android version [here](http://about.mattermost.com/mattermost-android-app). Source code can be found at the [GitHub Mattermost Mobile app repository](https://github.com/mattermost/mattermost-mobile).
```

Reply:
@justinegeffen please don't change the link format.. using markdown will open the links on the same window/tab the way  they are done with the anchors it opens in a new window/tab, that is done on purpose.